src/infrastructure/validation/testing/ITest.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * @author - Abhishek Saran(111701001)
+ * @module - Infrastucture module 
+ * @team - Test Harness
+ * @description - ITest interface with abstract class TestCase 
+ * @summary - Modules have to extend the abstract class to implement the run method for each test case  
+ */
+
+
+interface ITest{
+
+	//Methods
+	void setDescription(String des);
+	String getDescription();
+	void setError(String err);
+	String getError();
+	void setCategory();
+	String getCategory(String c);
+	void setPriority(int p);
+	String getPriority();
```

Comment:
Arrange these methods with getters coming first and then setters. Its easier to read. Give the full name to the parameters too